docs/src/browsers.md

```diff
@@ -3,35 +3,307 @@ id: browsers
+
+```bash js
+npx playwright install
+```
+
+```bash java
+mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="install"
+```
+
+```bash python
+playwright install
+```
+
+```bash csharp
+pwsh bin/Debug/netX/playwright.ps1 install
+```
+
+You can also install specific browsers by providing an argument:
+
+```bash js
+npx playwright install webkit
+```
+
+```bash java
+mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="install webkit"
+```
+
+```bash python
+playwright install webkit
+```
+
+```bash csharp
+pwsh bin/Debug/netX/playwright.ps1 install webkit
+```
+
+See all supported browsers:
+
+```bash js
+npx playwright install --help
+```
+
+```bash java
+mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="install --help"
+```
+
+```bash python
+playwright install --help
+```
+
+```bash csharp
+pwsh bin/Debug/netX/playwright.ps1 install --help
+```
+
+## Install browsers via API
+* langs: csharp
+
+It's possible to run [Command line tools](./cli.md) commands via the .NET API:
+
+```csharp
+var exitCode = Microsoft.Playwright.Program.Main(new[] {"install"});
+if (exitCode != 0)
+{
+    throw new Exception($"Playwright exited with code {exitCode}");
+}
+```
+
+## Install system dependencies
+
+System dependencies can get installed automatically. This is useful for CI environments.
+
+```bash js
+npx playwright install-deps
+```
+
+```bash java
+mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="install-deps"
+```
+
+```bash python
+playwright install-deps
+```
+
+```bash csharp
+pwsh bin/Debug/netX/playwright.ps1 install-deps
+```
+
+You can also install the dependencies for a single browser only by passing it as an argument:
+
+```bash js
+npx playwright install-deps chromium
+```
+
+```bash java
+mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="install-deps chromium"
+```
+
+```bash python
+playwright install-deps chromium
+```
+
+```bash csharp
+pwsh bin/Debug/netX/playwright.ps1 install-deps chromium
+```
+
+It's also possible to combine `install-deps` with `install` and install by that the browsers and OS dependencies with a single command. This would do both for Chromium, but you can also leave it out.
+
+```bash js
+npx playwright install --with-deps chromium
+```
+
+```bash java
+mvn exec:java -e -D exec.mainClass=com.microsoft.playwright.CLI -D exec.args="install --with-deps chromium"
+```
+
+```bash python
+playwright install --with-deps chromium
+```
+
+```bash csharp
+pwsh bin/Debug/netX/playwright.ps1 install --with-deps chromium
+```
+## Browser Configuration
+* langs: js
+
+Playwright Test supports multiple "projects" so that can run your tests in multiple browsers and configurations. Here is an example that runs every test in Chromium, Firefox and WebKit, by creating a project for each.
```

Comment:
```suggestion
Playwright supports multiple "projects" so that can run your tests in multiple browsers and configurations. Here is an example that runs every test in Chromium, Firefox and WebKit, by creating a project for each. You can also add branded browsers and mobile devices as "projects", see below.
```